framework/Models/EnsembleModel.py

```diff
@@ -432,69 +432,36 @@ def collectOutput(self,finishedJob,output):
     evaluation = finishedJob.getEvaluation()
     if isinstance(evaluation, Runners.Error):
       self.raiseAnError(RuntimeError,"Job " + finishedJob.identifier +" failed!")
-    out = evaluation[1]
-    exportDict = {'inputSpaceParams':{},'outputSpaceParams':{},'metadata':{}}
-    exportDictTargetEvaluation = {}
-    outcomes, targetEvaluations, optionalOutputs = out
+    outcomes, targetEvaluations, optionalOutputs = evaluation[1]
     try:
       jobIndex = self.tempOutputs['uncollectedJobIds'].index(finishedJob.identifier)
       self.tempOutputs['uncollectedJobIds'].pop(jobIndex)
     except ValueError:
       jobIndex = None
+
+    #joinedGeneralMetadata = {}
```

Comment:
remove

Reply:
This is a placeholder for me now to remember to readd the general metadata